src/applications/vaos/components/calendar/CalendarCell.jsx

```diff
@@ -33,6 +36,22 @@ const CalendarCell = ({
     'vaos-calendar__cell-selected': inSelectedArray,
   });
 
+  let selectedIndicator = <i className="fas fa-check vads-u-color--white" />;
```

Comment:
You might consider moving this into the `CalendarSelectedIndicatorBubbles` component you created (and removing the `Bubbles` part of the name). That way the `CalendarCell` only cares about whether the indicator is shown or not and the `CalendarSelectedIndicator` has all the logic for what specific indicator to show.